Doc/tutorial/errors.rst

```diff
@@ -409,6 +409,16 @@ For example::
    >>> bool_return()
    False
 
+   >>> def bool_return():
+   ...    try:
+   ...        return print('try return statement evaluated')
+   ...    finally:
+   ...        return False
+   ...
+   >>> bool_return()
+   try return statement evaluated  # None from print() not returned.
```

Comment:
I think it's better if there is an explicit "return 42" there rather than assuming people reading the tutorial know that print returns None.

Reply:
I agree, I've though about this a couple of times in the last week or two. Returning 42 doesn't satisfy what is being demonstrated, namely, that the statement (be it `print('try return statement evaluated')` or `42`) in the `try` clause *is evaluated* even though the value is not returned. 

I've thought about a few options, but they all seem to add complexity:
```python
>>> def func():
...     try:
...         print(42)
...         return print(49)
...     finally:
...         return print(59)
... 
>>> func()
42
49
59
>>> print(func())
42
49
59
None
```
...shows that the `try` code is evaluated (because 42 is printed), and the `try` `return` statement runs (printing 49), but not returned (because `None` is returned by the `finally`, not between 49 and 59). 
I'm just struggling to make that clear without `print` - is there another easily understandable side effect that can be put in the `try`'s `return` to demonstrate that it runs but is not returned?

Reply:
You could consider defining a function that prints and returns something non-None, then using that instead of `print`:

```
>>> def return_true():
...     print('Returning True')
...     return True
... 
>>> def return_bool():
...     try:
...         return return_true()
...     finally:
...         return False
... 
>>> return_bool()
Returning True
False
```